zipline/finance/risk/period.py

```diff
@@ -100,12 +100,9 @@ def calculate_metrics(self):
         self.num_trading_days = len(self.benchmark_returns)
         self.trading_day_counts = pd.stats.moments.rolling_count(
             self.algorithm_returns, self.num_trading_days)
-        self.mean_algorithm_returns = pd.Series(
-            index=self.algorithm_returns.index)
-        for dt, ret in self.algorithm_returns.iteritems():
-            self.mean_algorithm_returns[dt] = (
-                self.algorithm_returns[:dt].sum() /
-                self.trading_day_counts[dt])
+
+        self.mean_algorithm_returns = \
```

Comment:
The first element of the vectors `algorithm_returns` and `mean_algorithm_returns` are equal, i.e. they are both the return of the first day.

That behavior is shared both on existing behavior and with this patch which uses `cumsum`.

Quick snippet to confirm the `cumsum` behavior http://docs.scipy.org/doc/numpy/reference/generated/numpy.cumsum.html
```
In [2]: np.cumsum([0.001, 0.002, 0.003])
Out[2]: array([ 0.001,  0.003,  0.006])
```